aws-lambdas/iac/terraform/main.tf

```diff
@@ -305,3 +330,70 @@ resource "aws_cloudwatch_event_target" "invoke_lambda" {
   rule = aws_cloudwatch_event_rule.lambda_cron.name
   arn  = module.cron.lambda_arn
 }
+
+################################################################################
+## elasticache-redis
+################################################################################
+
+module "vpc" {
+  count   = var.custom_ec_vpc != {} ? 0 : 1
+  source  = "cloudposse/vpc/aws"
+  version = "2.0.0"
+
+  ipv4_primary_cidr_block = "172.16.0.0/16"
+
+  context = {
+    tags = tomap({
+      "Name"    = "arc-lambda-boilerplate-vpc",
+      "Project" = "arc-lambda-boilerplate"
+    })
+  }
+}
+
+module "ec-subnets" {
+  count   = var.custom_ec_vpc != {} ? 0 : 1
+  source  = "cloudposse/dynamic-subnets/aws"
+  version = "2.0.4"
+
+  availability_zones   = var.ec_availability_zones
+  vpc_id               = module.vpc[0].vpc_id
+  igw_id               = module.vpc[0].igw_id
+  ipv4_cidr_block      = module.vpc[0].vpc_cidr_block
+  nat_gateway_enabled  = true
+  nat_instance_enabled = false
+
+  context = {
+    tags = tomap({
+      "Name"    = "arc-lambda-boilerplate-vpc-subnet",
+      "Project" = "arc-lambda-boilerplate"
+    })
+  }
+}
+
+module "redis" {
+  count   = var.custom_ec_vpc != {} ? 0 : 1
+  source  = "cloudposse/elasticache-redis/aws"
+  version = "0.49.0"
+
+  availability_zones         = var.ec_availability_zones
+  vpc_id                     = module.vpc[0].vpc_id
+  allowed_security_group_ids = [module.vpc[0].vpc_default_security_group_id]
+  subnets                    = module.ec-subnets[0].private_subnet_ids
+  apply_immediately          = true
+  automatic_failover_enabled = false
+  at_rest_encryption_enabled = true
+  transit_encryption_enabled = true
+  parameter = [
+    {
+      name  = "notify-keyspace-events"
+      value = "lK"
+    }
+  ]
+
+  context = {
```

Comment:
Change `context` to `tags = module.tags.tags`

Reply:
not relevant anymore